File: api/HIN_API/Models/OpportunityAcquisitionCost.cs
using Microsoft.OData.Edm;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace HIN_API.Models
{
    public class OpportunityAcquisitionCost
    {
        public List<Users> UserList { get; set; }
        public List<Deal> OpportunityList { get; set; }
        public int? OpportunityStatusId { get; set; }
        public DateTime? OppCreatedFrom { get; set; }
        public DateTime? OppCreatedTo { get; set; }
        public int? OppClosingPercentFrom { get; set; }
        public int? OppClosingPercentTo { get; set; }
        public int? OppFunnelPercentFrom { get; set; }
        public int? OppFunnelPercentTo { get; set; }
        public DateTime? OppEstimateFrom { get; set; }
        public DateTime? OppEstimateTo { get; set; }
        public DateTime? From { get; set; }
        public DateTime? To { get; set; }
        public int? OppActualAmountFrom { get; set; }
        public int? OppActualAmountTo { get; set; }
        
    }
}

File: webapp/HIN_WEB/ClientApp/src/app/model/opportunityacquisitioncost.ts
import { Users } from "../users/users";
import { Deal } from "./deal";

export class Opportunityacquisitioncost {
  public UserList: Users;
  public OpportunityList: Deal;
  public OpportunityStatusId: number;
  public OppCreatedFrom: Date;
  public OppCreatedTo: Date;
  public OppClosingPercentFrom: number;
  public OppClosingPercentTo: number;
  public OppFunnelPercentFrom: number;
  public OppFunnelPercentTo: number;
  public OppEstimateFrom: Date;
  public OppEstimateTo: Date;
  public From: Date;
  public To: Date;
  public OppActualAmountFrom: number;
  public OppActualAmountTo: number;
}

